packages/react-examples/README.md

```diff
@@ -3,3 +3,71 @@
 **Examples and documentation for [Fluent UI React](https://developer.microsoft.com/en-us/fluentui)**
 
 This package is only meant to be consumed by variants of the Fluent UI documentation site and therefore has **NO GUARANTEE OF API STABILITY**.
+
+## Architecture
+
+This package contains various root folders within `src`
+
+```sh
+|- src/
+|--- e2e/
+|--- azure-themes/
+|--- react/
+|--- react-cards/
+|--- react-charting/
+|--- react-experiments/
+|--- react-focus/
+```
+
+1. `e2e/` contains helpers for e2e ? TODO
+2. rest of the folders contain particular package `stories` and documentation content for `public-docsite` and `public-docsite-resources`
```

Comment:
I'm not sure we really need a list of all the folders--seems like the type of thing that just gets out of date, though granted in this case it's less likely to change.

Also, the e2e folder can be deleted after Ling's PR to switch to cypress component testing.

```suggestion
Each folder under `src` is the unscoped name of a package, containing examples for that package.

The examples under `react` (plus `react-focus` and `react-cards`) are used on the [public documentation site](https://developer.microsoft.com/fluentui) via `apps/public-docsite`, and [aka.ms/fluentdemo](https://aka.ms/fluentdemo) via `apps/public-docsite-resources`.

The other packages' examples are only used for local development and on the [PR deploy site](https://fluentuipr.z22.web.core.windows.net/heads/master/).
```

Reply:
the list of folders was added because that weird `e2e` and also because there won't be any new features being added to v8 thus no change